### ISSUE TITLE
guides and links from the app does not open in browser

### ISSUE THREAD

**Issue by Potatovibin** [Marekkon5/onetagger#425]
This is especially annoying when i click login in the spotify client id/ client secret section and it doesn't open the website or when i can't check the guide which shows how to do the spotify thing

**Comment by Marekkon5** [comment]
Hello, which OS and which version of 1T? Could you perhaps try the latest Actions build? It might have some fixes. Thanks

**Comment by Potatovibin** [comment]
OS is Windows 11 Pro
where can i find 1T version

**Comment by Potatovibin** [comment]
Found the solution 

I use the Arc browser and for some reason it doesn't open in arc

**Comment by Marekkon5** [comment]
Not to shift the blame, but 1T just tells the OS to open in default browser, so maybe the issue is not 1T. Closing for now. You can test the latest Actions build, maybe it will work. However feel free to comment or open new issues if you have more problems. Thanks

**Comment by Potatovibin** [comment]
It could be because arc dosen't automatically open a tab? (most browsers have a startup page tag while in arc it's just empty)

**Comment by Potatovibin** [comment]
Where can i test the latest action build? (is it like nightly builds?)

**Comment by Marekkon5** [comment]
Yes, you can find it in the Actions tab on top (on Github) and then click the first one with green tick, and on the bottom there should be downloads for all platforms.

**Comment by Potatovibin** [comment]
Thanks for both the info and the quick reply!
